src/components/breadcrumbs/breadcrumbs.types.ts

```diff
@@ -0,0 +1,27 @@
+import { Link } from '@limetech/lime-elements';
+import { Icon } from 'src/global/shared-types/icon.types';
```

Comment:
@jgroth suggested that this import should be like this:
`import { Link, Icon } from '@limetech/lime-elements';`

But that generates this error:
<img width="679" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/f23a91ff-9142-46c3-a280-8895f6fa2b72">

I don't really know how it's possible to write `import { Link } from '@limetech/lime-elements';`. Because that is removed from exports here: https://github.com/Lundalogik/lime-elements/commit/60578ba0501c9a9cd8d0de10bd1bbfbc207c49af#diff-4f3136d98245087ba43f112ee10bfb0fcc0147dff7afa6a36c8f3f61555821cbL2

This is anyhow a discussion for the engineers. I leave it like this for now

Reply:
Importing from `@limetech/lime-elements` is more important in the examples so that they look correct from a consumer perspective. 

`Icon` is not exported from there yet I think because it generated some build error, probably the same kind of error we got when working on #2370

Reply:
I see. But we don't import in examples. So this should be alright then. If something should happen regarding the export of `Link` and `Icon`, I suspect you guys will do it elsewhere.